src/app.js

```diff
@@ -1,13 +1,186 @@
 // Vendor Modules
 import $ from 'jquery';
 import _ from 'underscore';
+import 'jquery-modal';
 
 // CSS
 import './css/foundation.css';
 import './css/style.css';
 
-console.log('it loaded!');
+// Import Trip Model and Collection
+import Trip from './app/models/trip';
+import TripList from './app/collections/trip_list';
+import Reservation from './app/models/reservation';
+
+const tripFields = ['name', 'continent', 'about', 'category', 'weeks', 'cost'];
+const reservationFields = ['name', 'age', 'email'];
```

Comment:
Since we're also using these variables in `models/trip.js` and `models/reservation.js`, we should probably make them into constants on the models themselves:

```javascript
// in models/trip.js

const Trip = Backbone.Model.extend({
  // Trip model stuff
});

Trip.FIELDS = ['name', 'continent', 'about', 'category', 'weeks', 'cost'];
```